runtime/include/sandbox_functions.h

```diff
@@ -28,6 +28,19 @@ sandbox_close_http(struct sandbox *sandbox)
 	client_socket_close(sandbox->client_socket_descriptor, &sandbox->client_address);
 }
 
+/**
+ * Remove the client fd from epoll
+ */
+static inline void
```

Comment:
I don't understand the code performance for the fastpath here. My understanding for the inline is it won't jump to the code but it has the whole block of the code in there because the code has been copied. If it can be inline, the compiler will do it; if it cannot, the compiler will ignore it. I add `inline` because I saw all functions in this file has inline, but I don't know how it is related to the fastpath, could you explain more about the fastpath?

Reply:
The function likely shouldn't be in this file, and instead in a .c file with the rest of the functions that aren't as performance sensitive.

Reply:
I agree. I don't know why these epoll operation was set an `inline` function. There is no corresponding .c file for this header file. If this function should be put into a .c file, I think the other functions in this file, such as `sandbox_open_http` and `sandbox_close_http` should also be moved into the same .c file.